src/chapters/enums/exercises.rst

```diff
@@ -0,0 +1,63 @@
+Exercises: Enum Practice
+========================
+
+#. Fork and clone `enumerable-planets <https://github.com/LaunchCodeEducation/enumerable-planets>`__.
+
+#. In the project, navigate to the data package.
+
+#. Create a new public enum called ``Planets``.
+
+   .. sourcecode:: java
+
+      public enum Planets {
+         // list the planets here.
+         // Mercury, Venus, Earth, Mars, Jupiter,
+         // Saturn, Uranus, Neptune
+         // Don't forget to capitalization convention and enum 
+         // syntax to separate value and end the list
+      }
+
+#. Before adding any other fields to ``Planets``, go to ``PlanetController``
+   and update the index handler to pass in a model argument. 
```

Comment:
I'd be more explicit here and reference the class `Model`, since it's confusing having two "models" floating around.